nifi-registry-core/nifi-registry-web-api/src/main/resources/templates/index.html.hbs

```diff
@@ -477,6 +481,38 @@
             </div>
             <div id="item-endpoints" class="section-endpoints hidden"></div>
         </div>
+        <div class="section">
+            <div class="section-header">
+                <div class="title link">Flows</div>
+                <div class="sub-title section-description">Flow endpoints</div>
+                <div class="clear"></div>
+            </div>
+            <div id="flow-endpoints" class="section-endpoints hidden"></div>
+        </div>
+        <div class="section">
+            <div class="section-header">
+                <div class="title link">Bundles</div>
+                <div class="sub-title section-description">Bundle endpoints</div>
+                <div class="clear"></div>
+            </div>
+            <div id="bundle-endpoints" class="section-endpoints hidden"></div>
+        </div>
+        <div class="section">
+            <div class="section-header">
+                <div class="title link">Extensions</div>
+                <div class="sub-title section-description">Extension endpoints</div>
+                <div class="clear"></div>
+            </div>
+            <div id="extension-endpoints" class="section-endpoints hidden"></div>
+        </div>
+        <div class="section">
+            <div class="section-header">
+                <div class="title link">Extension Repository</div>
+                <div class="sub-title section-description">Extension Repository endpoints</div>
+                <div class="clear"></div>
+            </div>
+            <div id="extension-repository-endpoints" class="section-endpoints hidden"></div>
+        </div>
```

Comment:
Are we keeping the Handlebars-based REST API documentation in addition to the new ascii-doc based HTML, or can this go away entirely?

Reply:
I figured it didn't hurt to leave the old approach around for at least one release in case we needed to switch back for some reason. With everything as is, all we would have to do is switch the link on the left side of the main documentation.jsp to point at the old index.html rather than the new rest-api.html.